docs/specification/current.md

```diff
@@ -1004,7 +1004,7 @@ CDS Services and browser-based CDS Clients will require CORS support. A secure i
 
 In the case that CDS Hooks cards are persisted, clients should take care to ensure that stale guidance does not negatively impact patient care.
 
-Services can update their guidance by returning a new set of `cards` when the service is invoked based on a different `hook`. Services indicate this intend by providing multiple CDS Services with the same `id` in [discovery](#discovery). Clients are recommended to remove `cards` returned by a previous invocation with the new `cards`. 
+CDS Services can then update their guidance by returning a new set of `cards` when the service is invoked based on a different `hook`. CDS Services indicate this intend by providing multiple CDS Services with the same `id` in [discovery](#discovery). Clients are recommended to remove `cards` returned by a previous invocation with the new `cards`. 
```

Comment:
```suggestion
CDS Services can update their previously returned guidance by returning a new set of `cards` when the service is invoked based on a different `hook`. CDS Services indicate this intent by providing multiple CDS Services with the same `id` in [discovery](#discovery). Clients are recommended to remove `cards` returned by a previous invocation with the new `cards`. 
```

Using "then" to try to refer to a process described in a previous paragraph is awkward. By explaining that guidance was previously returned, this paragraph remains independently meaningful. 

If you really dislike this fix, the other option would be merge the previous paragraph into this one to make the word "then" more meaningful.